assume/world.py

```diff
@@ -479,6 +480,14 @@ def create_unit(
             **unit_params,
         )
 
+    def _add_simulation_horizon_to_learning_role(self):
+        """
+        Add simulation horizon to learning role for calculation of decay.
+        """
+
+        self.learning_role.start = datetime2timestamp(self.start)
+        self.learning_role.end = datetime2timestamp(self.end)
+
```

Comment:
Why did you decide to make this a function of world and not the learning_role?

Reply:
I think I just tried to keep the structure of the function below __add_bidding_strategies_to_learning_role_ 
But I see, it would probably be better as a setter function of the LearningRole with start, end passed as args.